en/pe-docs/docs/devops/automation-pipelines/inbuilt-templates.md

```
@@ -0,0 +1,80 @@
+[Choreo’s pipeline specification](https://github.com/wso2/choreo-pipeline-specification/tree/main/docs/specification) supports referencing these templates using the following syntax:
+
+```
+steps:
+  - name: <step name>
+    template: choreo/<template name>
+```
+
+## Available Built-in Templates
+
+!!! info "Note"
+    The list of built-in templates is continuously growing. Check list of Choreo templates in [here](https://github.com/wso2/choreo-pipeline-specification/blob/main/docs/specification/built-in-templates.md)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)